quote-server/server.js

```diff
@@ -2,31 +2,55 @@
 // This is where your node app starts
 
 //load the 'express' module which makes writing webservers easy
-import express from "express";
+import express, { response } from "express";
 //load the quotes JSON
-import quotes from "./quotes.json" assert { type: "json" };
+// import quotes from "./quotes.json" assert { type: "json" };
 
 const app = express();
-// Now register handlers for some routes:
-//   /                  - Return some helpful welcome info (text)
-//   /quotes            - Should return all quotes (json)
-//   /quotes/random     - Should return ONE quote (json)
-app.get("/", (request, response) => {
-  response.send("Neill's Quote Server!  Ask me for /quotes/random, or /quotes");
+
+const getQuotes = async (url) => {
+  const response = await fetch(url);
+  return response.json();
+};
+
+app.get("/", async (request, response) => {
+  response.send("Marcus' Quote Server!  Ask me for /quotes/random, or /quotes");
 });
 
-//START OF YOUR CODE...
+app.get("/quotes", async (req, res) => {
+  const pageNumber = req.query.page;
+  const getAllQuotes = await getQuotes(
+    `https://api.quotable.io/quotes?page=${pageNumber}`
+  );
+  res.send(getAllQuotes);
+});
 
-//...END OF YOUR CODE
+app.get("/quotes/random", async (req, res) => {
+  const pageNumber = req.query.page;
+  const getAllQuotes = await getQuotes(
+    `https://api.quotable.io/quotes?page=${pageNumber}`
+  );
+  const randomQuotes =
+    getAllQuotes.results[
+      Math.floor(Math.random() * getAllQuotes.results.length)
```

Comment:
You will get a security issue raised using `Math.Random()` due to reasons raised in this [article](https://kemilbeltre.medium.com/why-do-not-use-math-random-a6f8b0ad38dd). 

This won't be an issue for this work, but it is worth being aware of in the future.